examples/website-demos/social-messaging/src/components/MessageList/SocialMessageList.tsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 
+import { MessageList } from 'stream-chat-react';
+
 import './SocialMessageList.scss';
 
 type Props = {};
 
 export const SocialMessageList: React.FC<Props> = (props) => {
```

Comment:
i see this just a placeholder but we shouldn't need a message list wrapper component, unless there's an upcoming feature i'm missing

Reply:
You're not missing anything, that was just a placeholder initially. Will remove!